app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Tuition fee and maintenance loans"
+heading: "Tuition fee and maintenance loans"
+description: |-
+    Find out which loans you could get to help pay your fees and living expenses while you train.
+related_content:
+    Who do you want to teach?: "/train-to-be-a-teacher/who-do-you-want-to-teach"
+    Improve your subject knowledge: "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    Salaries and benefits: "/salaries-and-benefits"
+promo_content:
+    - content/funding-and-support/promos/get-adviser-funding-promo
+calls_to_action:
+    student-finance-calculator:
+      name: simple
+      arguments:
+        icon: "icon-arrow"
+        text: "Use the student finance calculator on GOV.UK to find out how much funding you can get."
+        link_text: "Student finance calculator"
+        link_target: "https://www.gov.uk/student-finance-calculator"
+navigation: 20.10
+navigation_description: Find out how much money you could get to help pay your fees and living expenses while you train.
+keywords:
+    - Tuition Fee
+    - Maintenance Loan
+    - Student Loan
+    - Student Finance
+    - Financial Support
+---
+
+Funding is available to help you train to teach. You can apply for a:
+
+- tuition fee loan of up to £9,250 to cover your teacher training, so you do not need to pay course fees upfront
+- maintenance loan of up to £12,667 to help with living costs
+
+You can still apply for a tuition fee and a maintenance loan if you already have a student loan, and regardless of whether you get a teaching bursary or scholarship.
+
+You will only have to make loan repayments once you’re earning. Your repayments will not increase if you already have a student loan and take an additional loan for teacher training.
+
+$student-finance-calculator$
+
+## Check your eligibility for a scholarship or bursary
+
+You may also be eligible for a scholarship or bursary, depending on the subject you're training to teach.
+
+You do not have to pay a scholarship or bursary back. You'll receive this as well as your tuition fee and maintenance loans.
```

Comment:
@EllieNodder is it worth rewording this para to make it a little clearer? First sentence is fine. Change second to something like: 'You can receive this as well as any tuition fee and maintenance loans'.